CHANGES.rst

```diff
@@ -1,6 +1,270 @@
 Traits CHANGELOG
 ================
 
+Release 6.1.0
+-------------
+
+Released: 2020-05-XX
+
+The Traits library is a foundational component of the Enthought Tool Suite. It
+provides observable, typed attributes for Python classes, making those classes
+suitable for event-driven dataflow programming and for immediate use as models
+for graphical user interfaces, like those provided by the TraitsUI library.
+
+Traits 6.1 is the latest feature release in the Traits 6 series, and contains
+several major improvements.
+
+Highlights of this release
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* A new :mod:`observation <traits.observation>` framework for observing traited
+  attributes and other observable objects has been introduced. This is intended
+  to provide a full replacement for the existing :func:`on_trait_change`
+  mechanism, and aims to fix a number of fundamental flaws and limitations of
+  that mechanism. See the :ref:`observation <observe-notification>` section of
+  the user manual for an introduction to this framework.
+
+* New :class:`~traits.trait_list_object.TraitList`,
+  :class:`~traits.trait_dict_object.TraitDict` and
+  :class:`~traits.trait_set_object.TraitSet` classes have been added,
+  subclassing Python's built-in ``list``, ``dict`` and ``set``
+  (respectively).
+  Instances of these classes are observable objects in their own right, and
+  it's possible to attach observers to them directly. These classes were
+  primarily introduced to support the new observation framework, and are not
+  expected to be used directly. The API for these objects and their
+  notification system is provisional, and may change in a future Traits
+  release.
+
+* A new :class:`.Union` trait type has been added. This is intended as a simpler
+  replacement for the existing :class:`.Either` trait type. The :class:`.Either`
+  trait type may eventually be deprecated: users are encouraged to replace uses
+  of :class:`.Either` with :class:`.Union` where possible.
+
+* New :class:`.PrefixList`, :class:`.PrefixMap` and :class:`.Map` trait types
+  have been added. These replace the existing :class:`.TraitPrefixList`,
+  :class:`.TraitPrefixMap` and :class:`.TraitMap` subclasses of
+  :class:`.TraitHandler`, which are deprecated.
+
+* ``mypy`` typing stubs for the Traits library have been added in a separate
```

Comment:
This isn't right - it unnecessarily highlights `mypy`, when the feature is about typing support generally and not `mypy` support in particular. I'll reword.

Reply:
Fixed.